docs/developer-guide/advanced-project-customization.md

```diff
@@ -0,0 +1,222 @@
+
+> java -Ddatadir.location=/etc/mapstore/datadir
+
+This command will make the contents of the datadir available for the application **in runtime**. For further details on setting up and usage of the `datadir`, refer to the section [using a data directory](https://docs.mapstore.geosolutionsgroup.com/en/latest/developer-guide/externalized-configuration/#using-a-data-directory).
+
+Alternatively, for development purposes only, you can set the same property by modifying the `web/pom.xml` file inside your custom project.
+
+Find the entry for `cargo-maven3-plugin` and add the `systemProperties` tag within the `container` tag, so that your configuration looks like this:
+
+```xml
+        <plugin>
+            <groupId>org.codehaus.cargo</groupId>
+            <artifactId>cargo-maven3-plugin</artifactId>
+            <configuration>
+                <container>
+                    <containerId>tomcat8x</containerId>
+                    <systemProperties>
+                        <datadir.location>[local_path_to_your_datadir]/datadir/[customInstallationFolder]</datadir.location>
+                    </systemProperties>
+                    <zipUrlInstaller>
+                        <url>https://repo.maven.apache.org/maven2/org/apache/tomcat/tomcat/8.5.69/tomcat-8.5.69.zip</url>
+                    </zipUrlInstaller>
+                </container>
+                <configuration>
+                    <type>standalone</type>
+                    <home>
+                        ${project.build.directory}/apache-tomcat-${tomcat.version}
+                    </home>
+                    <properties>
+                        <cargo.servlet.port>${tomcat.port}</cargo.servlet.port>
+                        <cargo.logging>low</cargo.logging>
+                    </properties>
+                </configuration>
+                <deployables>
+                    <deployable>
+                        <groupId>${project.groupId}</groupId>
+                        <artifactId>${project.artifactId}</artifactId>
+                        <type>war</type>
+                        <properties>
+                            <context>/mapstore</context>
+                        </properties>
+                    </deployable>
+                </deployables>
+            </configuration>
+        </plugin>
+```
+
+In order to setup the multiple installations we can, for example, create three folders inside our `datadir`, denoting three different installations.
+
+```text
+├── datadir
+    ├── customA
+    ├── customB
+    ├── customC
+```
+
+Please note that the only requirement regarding folder structure is that the `datadir` exists, any content inside is optional and can be structured freely.
+
+As a final step, we will re-set the `datadir.location` system property outlined above to point to a specific installation, such that:
+
+> java -Ddatadir.location=/etc/mapstore/datadir/customA
+
+## Customize confiuration, assets and translations
```

Comment:
I'm going to fix it also on master.
```suggestion
## Customize configuration, assets and translations
```

Reply:
Here the commit on master
[72634ae](https://github.com/geosolutions-it/MapStore2/commit/72634ae3967a30724381b53c7b12d49131013348)